addons/website/static/src/js/editor/widget_link.js

```diff
@@ -38,26 +44,54 @@ weWidgets.LinkTools.include({
             classes: {
                 "ui-autocomplete": 'o_website_ui_autocomplete'
             },
-        }
-        wUtils.autocompleteWithPages(this, this.$('input[name="url"]'), options);
+        };
+        this.$urlInput = this.$('input[name="url"]');
+        this.$('.o_we_link_action_change').toggle(this.$urlInput.val().includes('/web/'));
```

Comment:
There are many things which seem horrible with this code, this has to be refactored: here we update the UI on start while there seems to be dedicated methods to do that. Also the template the start rendered already is rendered according to link info... to be updated afterwards. Also, that check `.includes('/web/'` feels not robust (why not startsWith, etc) and seems also to be a duplicate of `isFileUploaded` (whose name I don't really understand, it does not say if the file is uploaded or not but rather if the link points to a file? :thinking: ). Anyways, this has to be entirely refactored, it's impossible to follow what's what.